bigtable/client/internal/instance_admin.h

```diff
@@ -74,6 +74,19 @@ class InstanceAdmin {
    */
   std::vector<::google::bigtable::admin::v2::Instance> ListInstances(
       grpc::Status& status);
+
+  /**
```

Comment:
Please remove this comment, we are putting all the doxygen documentation in the non-`noex` class.